lib/phraseapp-ruby.rb

```diff
@@ -322,29 +322,29 @@ module RequestParams
   #   A note to help you remember what the access is used for.
   # scopes::
   #   A list of scopes that the access can be used for.
-  class AuthorizationParams
-    attr_accessor :expires_at, :note, :scopes 
+  class AuthorizationParams < ::OpenStruct
 
     def expires_at=(val)
-      @expires_at = DateTime.parse(val)
+      super(DateTime.parse(val))
     end
 
     def note=(val)
-      @note = val
+      super(val)
     end
 
     def scopes=(val)
-      @scopes = val.split(',')
+      super(val.split(','))
     end
 
     def validate
-      if @note == nil || @note == "" 
+      
+      if note == nil || note == "" 
         raise PhraseApp::ParamsHelpers::ParamsValidationError.new("Required parameter \"note\" of \"AuthorizationParams\" not set")
       end
     end
 
     def to_h
-      { expires_at: @expires_at, note: @note, scopes: @scopes }
+      @table.dup
     end
```

Comment:
What's the point of this method when OpenStruct already gives you `#to_h`? Duplicating has no effect since OpenStruct creates the hash on demand.

```ruby
foo = OpenStruct(foo: 'bar')
foo.to_h.object_id == foo.to_h.object_id # => false